tests/src/system/rest/GoCLINginxTests.scala

```diff
@@ -37,7 +43,7 @@ class GoCLINginxTests extends FlatSpec with Matchers with RestUtil {
     val Architectures = List("386", "amd64")
     val ServiceURL = getServiceURL()
 
-    it should s"respond to all files in root directory" in {
+    it should s"respond to all files in root directory" taggedAs(crossCompileCLI) in {
```

Comment:
Any Scala suggestions on the best way to check the JSON to see if specific keys exist in the JSON index file?